include/lsst/daf/base/DateTime.h

```diff
@@ -63,15 +63,15 @@ class DateTime {
     explicit DateTime(double date, DateSystem system = MJD, Timescale scale = TAI);
     DateTime(int year, int month, int day, int hr, int min, int sec,
              Timescale scale = TAI);
-    explicit DateTime(std::string const& iso8601);
+    explicit DateTime(std::string const& iso8601, Timescale scale);
 
     long long nsecs(Timescale scale = TAI) const;
     double get(DateSystem system = MJD, Timescale scale = TAI) const;
-    std::string toString(void) const;
+    std::string toString(Timescale scale) const;
 
-    struct tm gmtime(void) const; // Always UTC
-    struct timespec timespec(void) const; // Always UTC
-    struct timeval timeval(void) const; // Always UTC
+    struct tm gmtime(Timescale time) const; // Always UTC
+    struct timespec timespec(Timescale time) const; // Always UTC
+    struct timeval timeval(Timescale time) const; // Always UTC
```

Comment:
The comments are confusing. What influence does the Timescale have exactly if the output is always UTC?